core/controllers/admin_test.py

```diff
@@ -3036,3 +3056,87 @@ def test_update_blog_post_with_correct_params(self) -> None:
                 'published_on': '05/09/2000'
             },
             csrf_token=csrf_token)
+
+
+class GenerateDummyBlogTest(test_utils.GenericTestBase):
+    """Tests Generate Dummy data"""
```

Comment:
"Tests the generation of dummy blog post data."

Reply:
Done.